src/support/variables/typography.scss

```diff
@@ -32,29 +32,28 @@ $lh-condensed: 1.25 !default;
 $lh-default: 1.5 !default;
 
 // Font stacks
-// stylelint-disable no-eol-whitespace
```

Comment:
@simurai Is this `no-eol-whitespace` stylelint issue no longer a problem with this custom stack of fonts?

Reply:
No, I think it was only needed for these white spaces on empty lines. E.g. https://github.com/primer/css/pull/1573/files#diff-34665bdb6ca6905a302f6c4da0ce1a94b8fc0122bfa312635fb0136565abef3dL40-R39